docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,158 @@
+3. The client library will edit its local state to reflect that a new update has been added to the local cache. It will also update the local state with the new `expo-manifest-filters` and `expo-server-defined-headers` fround in the response [headers](#manifest-response-headers).
+
+The primary consumers of this spec are Expo Application Services and organizations that wish to manage their own update server to satisfy internal requirements.
+
+## Manifest Request
+A conformant client library MUST make a GET request with the headers:
+
+1. `expo-platform`, to specify the platform type the client is running on.
+    * iOS MUST be `expo-platform: ios`.
+    * Android MUST be `expo-platform: android`.
+    * If it is not one of these platforms, the server SHOULD return a 400 or a 404
+2. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file. 
+3. Any headers stipulated by a previous responses' [server defined headers](#manifest-response-headers):
+
+A conformant client library SHOULD also send `accept: application/expo+json, application/json`. With the order following the prefence ordering for the accept header specified in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.3.2). But MUST send at least one of them, `accept: application/expo+json` or `accept: application/json`.
+
+Example:
+```
+accept: application/expo+json, application/json
+expo-platform: string
+expo-runtime-version: string
+```
+
+## Manifest Response
+
+A conformant server MUST return a response containing a [manifest body](#manifest-response-body) and [headers](#manifest-response-headers).
+
+The choice of manifest and headers are dependent on the values of the request headers. A conformant server MUST:
+
+* Return a manifest that describes the most recent update ordered by creation time satisfying all constraints imposed by the [request headers](#manifest-request).
+
+### Manifest Response Headers
+
+```
+expo-protocol-version: 0
+expo-sfv-version: 0
+expo-manifest-filters: &lt;expo-sfv&gt;
+expo-server-defined-headers: &lt;expo-sfv&gt;
+cache-control: &lt;*&gt;
+content-type: &lt;*&gt;
+```
+
+* `expo-protocol-version` describes the version of the protocol defined in this spec and MUST be `0`.
+* `expo-sfv-version`  MUST be `0`.
+* `expo-manifest-filters` is an [Expo SFV 0](expo-sfv-0.md) dictionary. It is used to filter updates stored by the client library by the `metadata` attribute found in the [manifest](#manifest-response-body). If a field is mentioned in the filter, the corresponding field in the metadata must either be missing or equal for it to pass. The client library MUST store the manifest filters until it is overwritten by a newer response.
```

Comment:
@esamelson can you confirm that this is the correct filtering process?

Reply:
Yes, sounds correct to me!